packages/specifications/src/projet/lauréat/mainLevéeGarantiesFinancières/stepDefinitions/helper.ts

```diff
@@ -0,0 +1,27 @@
+import { IdentifiantProjet } from '@potentiel-domain/common';
+
+const defaultMotif = 'projet-achevé';
+const defaultUtilisateur = 'porteur@test.test';
+const defaultDateDemande = '2024-05-01';
+export const defaultDateRejetOuAccord = '2024-06-01';
+
+type SetMainlevéeDataProps = {
+  motif?: string;
+  utilisateur?: string;
+  dateDemande?: string;
+  identifiantProjet: IdentifiantProjet.ValueType;
+};
```

Comment:
petite suggestion :)

```suggestion
const defaultDataProps = {
  motif: 'projet-achevé',
  utilisateur: 'porteur@test.test',
  dateDemande: '2024-05-01',
  dateRejetOuAccord = '2024-06-01',
};

type SetMainlevéeDataProps = Partial<typeof defaultDataProps> & {
  identifiantProjet: IdentifiantProjet.ValueType;
};
```